### DISCUSSION TITLE
Dependency on node_modules during prod build of next app

### DISCUSSION THREAD

**Discussion by writam-roy** [vercel/next.js#27505] [Help]
Hi I have a question around prod build of a next app,once I am done with prod build of my next app,and want to deploy it,is there any way where I can deploy or serve it without installing the dependencies mentioned on package.json after build,to reduce the image size?
Note: This is about custom deployment,not on Vercel

**Comment by s-mishin** [comment]
A similar question. Is it possible to build a project so that you don't have to do an NPM install.
When I built my express server, it was configured with a webpack
